### PR TITLE
`_SharedKeyDefault<some SharedKey>` → `(some SharedKey).Default`

### PR DESCRIPTION
**Pull request by stephencelis**
Small refactor/cleanup so long as older Swift compilers work with it.

### CODE REVIEW

File: Sources/Sharing/Internal/Reference.swift [hunk 1]
```diff
@@ -339,10 +339,6 @@ final class _ManagedReference<Key: SharedReaderKey>: Reference, Observable {
 }
 
 extension _ManagedReference: MutableReference, Equatable where Key: SharedKey {
-  var wrappedValue: Key.Value {
-    base.wrappedValue
-  }
-
```
Comment (stephencelis) [inline Sources/Sharing/Internal/Reference.swift:345]: Unrelated cleanup: this is already defined in the base class, so no need to duplicate that work in this extension.

**Review by mbrandonw — APPROVED** [review]
